cliff.toml

```
@@ -27,7 +27,7 @@ body = """
     {%- for commit in commits | unique(attribute="message") %}
         {%- if commit.scope -%}
         {% else -%}
-            - {{ commit.message | upper_first }} {% if commit.links %}({% for link in commit.links | unique(attribute="text") %}[{{ link.text }}]({{ link.href }}){% endfor -%}){% endif %} ([{{ commit.id | truncate(length=7, end="") }}]({{ commit.id }}))
+            - {{ commit.message | upper_first }} {% if commit.links %}({% for link in commit.links | unique(attribute="text") %}[{{ link.text }}]({{ link.href }}){% endfor -%}){% endif %} ([{{ commit.id | truncate(length=7, end="") }}](https://github.com/mendersoftware/app-update-module/commit/{{ commit.id }}))
```

@mzedel I don't like this one so much, however I cannot find a better way to generate links that could be used also from outside this repo (namely: docs.mender.io)

---

There should be https://git-cliff.org/docs/templating/context where we can get some info from, but I don't know what it would output in CI/ what to filter for (locally it shows the full path to the repo directory).

---

that should maybe get us to:
```suggestion
            - {{ commit.message | upper_first }} {% if commit.links %}({% for link in commit.links | unique(attribute="text") %}[{{ link.text }}]({{ link.href }}){% endfor -%}){% endif %} ([{{ commit.id | truncate(length=7, end="") }}](https://github.com/mendersoftware/{{ repository | split(pat="/") | last }}/commit/{{ commit.id }}))
```
- not sure if we can get the organization in any way from there... or we have to provide it from CI through the `--github-repo` flag: https://git-cliff.org/docs/configuration/remote/

---

The `{{ repository }}` seems a too unstable solution: in my env I'm using git worktrees directories, so I can see as: 
```
https://github.com/mendersoftware/MC-7477-app-update-module-bootstrap/commit/9c46cdba380b8959a7387c47298ad348a88f23a3
```
Maybe we should just print a placeholder and run `sed` on it in the pipeline, right after git-cliff.. Thus, this is not optimal...

---

hmm yeah the `<REPO>` placeholder approach is also what I found in the git-cliff issues... another alternative might be the Github integration (https://git-cliff.org/docs/integration/github#remote / with the `GITHUB_TOKEN` accessible and `{{ remote.github.owner }}/{{ remote.github.repo }}`) - this would allow us to also get to the PR for a change... 🤔

---

Nice one! Here's the new result with `(https://github.com/{{ remote.github.owner }}/{{ remote.github.repo }}/commit/{{ commit.id }}))`:

```
## 1.1.0 - 2024-09-18


### Features


- Oci images deep delta support([MEN-7033](https://northerntech.atlassian.net/browse/MEN-7033)) ([4c6e975](https://github.com/mendersoftware/app-update-module/commit/4c6e975ef6fa832982ded74c34b6ea243c0c2f17))  by @merlin-northern
- Automatic image extraction from docker-compose file ([9c46cdb](https://github.com/mendersoftware/app-update-module/commit/9c46cdba380b8959a7387c47298ad348a88f23a3))  by @GoethalsRobbe

```